guides/common/modules/ref_port-and-firewall-requirements.adoc

```diff
@@ -42,6 +42,7 @@ ifdef::katello,satellite,orcharhino[]
 | 443, 80 | TCP | HTTPS, HTTP | {SmartProxy} | Content Retrieval | Content
 | 443, 80 | TCP | HTTPS, HTTP | Client | Content Retrieval | Content
 | 1883 | TCP | MQTT | Client | Pull based REX (optional) | Content hosts for REX job notification (optional)
+| 9090 | TCP | HTTPS | Client | Pull based REX (optional) | Content hosts for REX job notification (optional)
```

Comment:
I'm thinking about how to best order the list. Right now, you're adding the new line right below the other REX requirement, and that makes sense on some level.

On the other hand, the table as a whole is currently ordered based on the port numbers -- which means that for consistency, your new line would work best grouped together with the other lines describing 9090. So after line 53?

Reply:
These comments apply to the other table too.

Reply:
Done and done :smile: